subsys/bluetooth/controller/ll_sw/ll_tx_pwr.c

```diff
@@ -18,9 +18,16 @@
 #include "pdu.h"
 
 #include "lll.h"
+#if defined(CONFIG_BT_CTLR_CONN_PROP_META)
+#include "lll_conn_ext.h"
+#endif /* CONFIG_BT_CTLR_CONN_PROP_META */
 #include "lll_conn.h"
 #include "ull_conn_internal.h"
 
+#ifndef RADIO_TXP_DEFAULT
```

Comment:
This seems to be an unrelated change. Move to a different PR?

Reply:
> This seems to be an unrelated change. Move to a different PR?

Moved to a separate commit. Necessary for compiling the feature.

Reply:
Move this to top level subsys/bluetooth/controller/hal/radio.h